layouts/partials/sidebar/left.html

```diff
@@ -62,15 +62,15 @@ <h2 class="site-description">{{ .Site.Params.sidebar.subtitle }}</h2>
         {{ range .Site.Menus.main }}
         {{ $active := or (eq $currentPage.Title .Name) (or ($currentPage.HasMenuCurrent "main" .) ($currentPage.IsMenuCurrent "main" .)) }}
         <li {{ if $active }} class='current' {{ end }}>
-            <a href='{{ .URL }}' {{ if eq .Params.newTab true }}target="_blank"{{ end }}>
+            <a href='{{ .URL | absLangURL }}' {{ if eq .Params.newTab true }}target="_blank"{{ end }}>
```

Comment:
`absLangURL` would create problem when the menu item is added through page's frontmatter. See https://github.com/CaiJimmy/hugo-theme-stack/issues/770#issuecomment-1678713221

Reply:
I see...
I couldn't think of another solution at the moment. Let me close the MR for now and think about it later.
Anyway, thanks for review.

Reply:
Thanks to you! 

And don't worry about it! It's still possible to localize the menu manually in the config file.